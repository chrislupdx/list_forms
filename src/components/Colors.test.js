import React from 'react';
import { shallow } from 'enzyme';
import Colors from './Colors';

describe('Colors component', () => {
  it('renders a color', () => {
    const colors = [
      { name: 'Red', rgb: { red: 255, green: 0, blue: 0 } },
      { name: 'Blue', rgb: { red: 0, green: 0, blue: 333 } },
    ];
    const wrapper = shallow(<Colors colors={colors} />);
    expect(wrapper).toMatchSnapshot();
  });
});
