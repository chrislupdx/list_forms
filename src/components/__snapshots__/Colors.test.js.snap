// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Colors component renders a colors 1`] = `
<u1>
  <li
    key="Red"
  >
    <Color
      hex="#FF0000"
      name="Red"
    />
  </li>
  <li
    key="Blue"
  >
    <Color
      hex="#0000FF"
      name="Blue"
    />
  </li>
</u1>
`;
